SB-Maps/SB-Maps/Supporting Files/Localization/InterfaceString.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  InterfaceString.swift
+//  Raven
+//
+//  Created by Kerry Knight on 9/28/16.
+//  Copyright © 2017 JOMO, Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct InterfaceString {
+    public struct Selection {
+        static let Work: String = NSLocalizedString("Go to Work", comment: "")
+        static let Vacation: String = NSLocalizedString("Go on Vacation", comment: "")
+    }
+    
+    public struct Map {
+        static let CurrentLocation: String = NSLocalizedString("Current Location", comment: "")
+    }
```

Comment:
It's a good habit and a lot easier to add localization as you go to projects vs. trying to do it all at the end.